src/common/ui-components/ListView/ListView.module.scss

```diff
@@ -2,6 +2,11 @@
   padding: 1rem;
 }
 
+.divider {
+  height: 1px;
+  background: black;
```

Comment:
I understand this is work in progress, but a quick note: magic number

Reply:
Fixed.